.gitignore

```diff
@@ -0,0 +1,17 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```

Comment:
@pscacchi i think when you did the rebase of your brach merge repeated rules, we can only maintenance one .idea/ rule to ignore all idea folder content